guide/14-deep-learning/address_standerdization_using_model_extension.ipynb

```diff
@@ -0,0 +1,967 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        # This block of the code will parse params from emd or from kwargs through python API
</pre><p> </p>

<pre class="ql-syntax" spellcheck="false">This block of the code will parse params from emd or kwargs passed from arcgis.learn python API
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2112/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Done</p>